core/src/main/scala/chisel3/internal/MonoConnect.scala

```diff
@@ -99,7 +99,7 @@ private[chisel3] object MonoConnect {
     sourceInfo:  SourceInfo,
     sink:        Data,
     source:      Data,
-    context_mod: RawModule
+    context_mod: BaseModule
```

Comment:
What's the purpose for this change? I'm fine with it, just curious.

Reply:
Class extends BaseModule and connections between Properties are legal within Classes.